huaweicloud/services/acceptance/dms/resource_huaweicloud_dms_rocketmq_instance_test.go

```diff
@@ -0,0 +1,155 @@
+		CheckDestroy:      rc.CheckResourceDestroy(),
+		Steps: []resource.TestStep{
+			{
+				Config: testDmsRocketMQInstance_basic(rName),
+				Check: resource.ComposeTestCheckFunc(
+					rc.CheckResourceExists(),
+					resource.TestCheckResourceAttr(resourceName, "name", rName),
+					resource.TestCheckResourceAttr(resourceName, "engine", "reliability"),
+					resource.TestCheckResourceAttr(resourceName, "engine_version", "4.8.0"),
+				),
+			},
+			{
+				Config: testDmsRocketMQInstance_basic_update(rName, updateName),
+				Check: resource.ComposeTestCheckFunc(
+					rc.CheckResourceExists(),
+					resource.TestCheckResourceAttr(resourceName, "name", updateName),
+					resource.TestCheckResourceAttr(resourceName, "engine", "reliability"),
+					resource.TestCheckResourceAttr(resourceName, "engine_version", "4.8.0"),
+				),
+			},
+			{
+				ResourceName:      resourceName,
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
+
+func testAccDmsRocketmqInstance_Base(rName string) string {
+	return fmt.Sprintf(`
+resource "huaweicloud_vpc" "test" {
+  name        = "%[1]s"
+  cidr        = "192.168.0.0/24"
+  description = "Test for DMS RocketMQ"
+}
+
+resource "huaweicloud_vpc_subnet" "test" {
+  name       = "%[1]s"
+  cidr       = "192.168.0.0/24"
+  gateway_ip = "192.168.0.1"
+  vpc_id     = huaweicloud_vpc.test.id
+}
+
+resource "huaweicloud_networking_secgroup" "test" {
+  name        = "%[1]s"
+  description = "secgroup for rocketmq"
+}
+
+data "huaweicloud_availability_zones" "test" {}
+`, rName)
+}
+
+func testDmsRocketMQInstance_basic(rName string) string {
+	return fmt.Sprintf(`
+%s
+
+resource "huaweicloud_dms_rocketmq_instance" "test" {
+  name                = "%s"
+  engine              = "reliability"
+  engine_version      = "4.8.0"
+  total_storage_space = 600
+  vpc_id              = huaweicloud_vpc.test.id
+  subnet_id           = huaweicloud_vpc_subnet.test.id
+  security_group_id   = huaweicloud_networking_secgroup.test.id
+  availability_zone   = [
+    data.huaweicloud_availability_zones.test.names[0]
+  ]
+  flavor_id           = "c6.4u8g.cluster"
+  storage_spec_code   = "dms.physical.storage.high.v2"
+}
+`, testAccDmsRocketmqInstance_Base(rName), rName)
+}
+
+func testDmsRocketMQInstance_basic_update(rName, updateName string) string {
+	return fmt.Sprintf(`
+%s
+
+resource "huaweicloud_dms_rocketmq_instance" "test" {
+  name                = "%s"
+  engine              = "reliability"
+  engine_version      = "4.8.0"
+  total_storage_space = 600
+  vpc_id              = huaweicloud_vpc.test.id
+  subnet_id           = huaweicloud_vpc_subnet.test.id
+  security_group_id   = huaweicloud_networking_secgroup.test.id
+  availability_zone   = [
+    data.huaweicloud_availability_zones.test.names[0]
+  ]
+  flavor_id           = "c6.4u8g.cluster"
+  storage_spec_code   = "dms.physical.storage.high.v2"
+}
+`, testAccDmsRocketmqInstance_Base(rName), updateName)
+}
```

Comment:
useless function
function `testDmsRocketMQInstance_basic` is enough

Reply:
done